mp/game/momentum/materials/particle/bendibeam.vmt

```diff
@@ -0,0 +1,15 @@
+"Sprite"
+{
+	"$basetexture" "particle/bendibeam"
+	"$additive" "1"
+	$translucent 1
+	"$vertexcolor"		1
+	"$nocull"			1
+	//"$spriteorientation" "parallel_upright"
+	//"$spriteorigin" "[ 0.50 0.50 ]"
+	"$spriterendermode" 5
+	$overbrightfactor 15
+}
+
+
+
```

Comment:
Make sure to use spaces in all your text files and added code files, and to remove extra whitespace.